application_handlers.go

```diff
@@ -141,16 +141,25 @@ func ApplicationEdit(c echo.Context) error {
 		return err
 	}
 
-	input := &m.ApplicationEditRequest{}
-	if err := c.Bind(input); err != nil {
-		return err
+	id, err := strconv.ParseInt(c.Param("id"), 10, 64)
+	if err != nil {
+		return util.NewErrBadRequest(err)
 	}
 
-	id, err := strconv.ParseInt(c.Param("id"), 10, 64)
+	isPaused, err := applicationDB.IsPaused(&id)
```

Comment:
Here when err is not nil, I believe it is not 400, but 500, so we can return only err

Reply:
Thank you for the comments but I ended up removing the `IsPaused` functions since they're no longer necessary on this PR. I need to fetch the resource from the DB to be able to update it in any case, so I can check if the `PausedAt` member is nil directly, without having to go to the database again.